cookbook/examples/apps/content_creator/app.py

```diff
@@ -0,0 +1,300 @@
+import datetime
+import os
+import time
+
+import streamlit as st
+from agno.models.mistral.mistral import MistralChat
+from agno.models.openai import OpenAIChat
+from agno.utils.log import logger
+from config import PostType
+from dotenv import load_dotenv
+from scheduler import schedule_and_publish
+from utils import about_widget, clear_generated_content, create_iso_date
+from workflow import ContentPlanningWorkflow
+
+load_dotenv()
+
+OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
```

Comment:
also these shouldn't be required here as the model already imports them in their class while initialization. So we should be able to remove this.